app/controllers/candidate_interface/decoupled_referees_controller.rb

```diff
@@ -0,0 +1,11 @@
+module CandidateInterface
+  class DecoupledRefereesController < CandidateInterfaceController
```

Comment:
The resource we're creating is a Reference, so maybe `DecoupledReferences`? Consistent with the way we're describing the feature too.

Reply:
Yea i agree with this 👍